tests/wdio/pageobjects/Setup.page.js

```diff
@@ -12,30 +12,24 @@ class SetupPage extends Page{
 
     }
 
-    async pause(is_human){
-        if(is_human)
-            await browser.pause(this.is_human_pause);
-    }
-
     async open()
     {
-        await this.pause(this.is_human);
-        await browser.maximizeWindow();
+        await browser.pause(this.is_human_pause)
         await super.open(this.params.page.url);
     }
 
     async installButtonAssert(data){
-        await expect(Sl.testid(data.element.install_button_testid)).toExist();
+        await expect(Sl.$(data.element.install_button)).toExist();
```

Comment:
@we-abhijeet-g001 why are we not using `testid`?

Reply:
Sir, I think Vikram sir, changed something with the branch in gitKraken, So my previous code in shown here. I will update the code and push it